### ISSUE TITLE
Unnessassary horizental scroll on website on mobile

### ISSUE THREAD

**Issue by julianvargkim** [Microsoft/TypeScript#29679]
Galaxy S9, Android 8, Firefox.

Entire website other than the sticky menu bar is movable by horizontal scroll (right side). Not a website-brekaing issue but definitely an easily visible bug.

**Comment by j-oliveras** [comment]
What has to do with typescript if a website is not correctly displayed?

This is most likely a css problem o a code error not a typescript compiler bug.

**Comment by issafram** [comment]
> @j-oliveras I guess this is about typescriptlang.org
> And since there is no repo for the website this is the only place to report such issues.

There isn't a repo for it being really cold where I'm at.  Can I post an issue in this repo? /s

**Comment by WORMSS** [comment]
I have a bit of an itch on my foot, and the doctor doesn't have a repo, this is the correct place to report it right?

**Comment by RyanCavanaugh** [comment]
Confirmed on my Pixel 2 

This actually is the right repo for website issues, for now 🤷‍♂️

**Comment by orta** [comment]
Yeah, I think having it here was reasonable - now the website has a public repo, so I'm moving this over to that repo 👍
